extension/zpagesextension/zpagesextension_test.go

```diff
@@ -51,7 +51,7 @@ func newZpagesTelemetrySettings() component.TelemetrySettings {
 
 func TestZPagesExtensionUsage(t *testing.T) {
 	cfg := &Config{
-		confighttp.ServerConfig{
+		ServerConfig: confighttp.ServerConfig{
```

Comment:
sure, but can those changes be done in a separate PR?

Reply:
The field seems needed to be named after I add an [additional field](https://github.com/open-telemetry/opentelemetry-collector/pull/11964/files#diff-72b14bc1a91e77c0a29c2f8724af0a35fd650bd85fec8e462723317dba4c82f2R19) to enable the expvar.

Reply:
Ah ok. No problem.